.env.sample

```diff
@@ -1,21 +1,21 @@
 BITCOIND_URL=
-EMILY_URL=
-MEMPOOL_API_URL=https://mempool.space/api
-PUBLIC_MEMPOOL_URL=https://mempool.space
+EMILY_URL=http://temp.sbtc-emily-dev.com
+MEMPOOL_API_URL=https://mempool.bitcoin.regtest.hiro.so/api
+PUBLIC_MEMPOOL_URL=https://mempool.bitcoin.regtest.hiro.so
 BITCOIN_RPC_USER_NAME=
 BITCOIN_RPC_PASSWORD=
-WALLET_NETWORK=
-SBTC_CONTRACT_DEPLOYER=
+WALLET_NETWORK=sbtcTestnet
+SBTC_CONTRACT_DEPLOYER=SN69P7RZRKK8ERQCCABHT2JWKB2S4DHH9H74231T
 BANNER_CONTENT=
-RECLAIM_LOCK_TIME=
+RECLAIM_LOCK_TIME=144
 NEXT_TELEMETRY_DISABLED=1
 SUPPORT_LINK=
 POLLING_INTERVAL=5000
-BRIDGE_APP_URL=https://app.stacks.co
-STACKS_API_URL=https://api.hiro.so
+BRIDGE_APP_URL=http://localhost:3000
+STACKS_API_URL=https://api.testnet.hiro.so
 # comma separated values style
 # withdrawals,reskin
-FEATURE_FLAGS=
+FEATURE_FLAGS=reskin,withdrawals
```

Comment:
`withdrawals` is no longer used, also as a "sample" env why not having this empty instead?

Reply:
makes sense

Reply:
It might be worth to document feature flags in the README too, actually

Reply:
Sorry for the confusion, but can we move the feature flag list to the readme, and keep here only the format help (ie, line 16 and 17)?

Reply:
done